### ISSUE TITLE
SQL Injection Vulnerability

### ISSUE THREAD

**Issue by olcayertas** [owncloud/android#2590]
We have following warning message in Google Play Console:

```
    Security alert

    Your app contains an SQL Injection Vulnerability. Please see this Google Help Centre article for details.

    Vulnerable classes:

        Lcom/gozen/cloud/android/providers/FileContentProvider;->query

    Please fix the issue before: 08/23/2019

    Affects APK version 20100201.
```

Is this fixed?

https://support.google.com/faqs/answer/7668308

**Comment by davigonz** [comment]
Yes, this was fixed with 2.10.1, can you recheck it please? Your versionCode (20100201) is different from the official one, right?

**Comment by olcayertas** [comment]
Yes, it is differend. Thank you very much.
